Exiled.API/Features/Effect.cs

```diff
@@ -15,7 +15,7 @@ namespace Exiled.API.Features
     using Exiled.API.Extensions;
 
     /// <summary>
-    /// Useful class to save effect configs in a cleaner way.
+    /// Useful class to handle effect statuses.
```

Comment:
It was created for better serialization in configs, so previous summary was more correct

Reply:
Reverted